config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     port: process.env.PORT || 5000,
 
     // Hostname of the Taitan instance, and whether to communicate over HTTPS with it
-    taitanUrl: process.env.TAITAN_URL || "https://taitan.datasektionen.se",
+    taitanUrl: process.env.TAITAN_URL || "http://styrdokument-taitan.mjukglass.datasektionen.se",
```

Comment:
I enabled https for styrdokument-taitan since I noticed that you used http here, and now this doesn't work correctly unless we change this to https since http connections will be redirected to https, and that redirects the client to the taitan page :^)

Reply:
Yeah, this is a mistake from my part, it was supposed to be https, sorry